### DISCUSSION TITLE
three js doesn't export

### DISCUSSION THREAD

**Discussion by dzakyabdurhmn** [community/community#108617] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I made a website using next js and three js when I run dev threejs it can render but when I build then I run start the 3d model doesn't render

this is the repository: https://github.com/dzakyabdurhmn/itskinaa
and I cloned from: https://github.com/craftzdog/craftzdog-homepage

**Comment by dzakyabdurhmn** [comment]
![image](https://github.com/community/community/assets/100127440/67db9daf-f522-423a-b6dd-506b169695ae)
build file

**Reply by Rimadaqch** [reply]
comment1

**Reply by dzakyabdurhmn** [reply]
What do you mean
